### PR TITLE
fix format

### PR DESCRIPTION
**Pull request by matrunych**

### CODE REVIEW

File: ss_admin_console/main/tests/classes/SetUtilsTest.cls [hunk 1]
```diff
@@ -46,10 +46,10 @@
         Set<Object> objs = new Set<Object>{1, 2, 3};
 
         Test.startTest();
-        Object result = new SetUtils().toSetOfType(field,objs);
+        Object result = SetUtils.toSetOfType(field,objs);
```
Comment (Taraass) [inline ss_admin_console/main/tests/classes/SetUtilsTest.cls:49]: the same here

File: ss_admin_console/main/tests/classes/SetUtilsTest.cls [hunk 2]
```diff
@@ -58,10 +58,10 @@
         Set<Object> objs = new Set<Object>{Date.newInstance(1960, 2, 17)};
 
         Test.startTest();
-        Object result = new SetUtils().toSetOfType(field,objs);
+        Object result = SetUtils.toSetOfType(field,objs);
```
Comment (Taraass) [inline ss_admin_console/main/tests/classes/SetUtilsTest.cls:61]: and here

File: ss_admin_console/main/tests/classes/SetUtilsTest.cls [hunk 3]
```diff
@@ -70,10 +70,10 @@
         Set<Object> objs = new Set<Object>{Datetime.newInstance(2020, 2, 17)};
 
         Test.startTest();
-        Object result = new SetUtils().toSetOfType(field,objs);
+        Object result = SetUtils.toSetOfType(field,objs);
```
Comment (Taraass) [inline ss_admin_console/main/tests/classes/SetUtilsTest.cls:73]: and here

File: ss_admin_console/main/tests/classes/SetUtilsTest.cls [hunk 4]
```diff
@@ -82,9 +82,9 @@
         Set<Object> objs = new Set<Object>{true};
 
         Test.startTest();
-        Object result = new SetUtils().toSetOfType(field,objs);
+        Object result = SetUtils.toSetOfType(field,objs);
```
Comment (Taraass) [inline ss_admin_console/main/tests/classes/SetUtilsTest.cls:85]: and heree

**Review by Taraass — CHANGES_REQUESTED** [review]

**Review by mykhaylo-partyka — APPROVED** [review]

**Pull request merged**